README.md

```
@@ -62,12 +70,322 @@ lola sync
+## Creating a Module
+
+### 1. Initialize
+
+```bash
+lola mod init my-skills
+cd my-skills
+```
+
+This creates:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)